src/main/java/uk/gov/ons/ctp/response/casesvc/service/impl/CaseServiceImpl.java

```diff
@@ -298,10 +297,11 @@ public void saveCaseIacAudit(final Case updatedCase) {
    * @param targetCase Case to update
    */
   private void replaceIAC(final Case targetCase) {
-    if (!internetAccessCodeSvcClientService.isIacActive(targetCase.getIac())) {
+    String iac = targetCase.getIac();
+    if (iac == null || !internetAccessCodeSvcClientService.isIacActive(iac)) {
       log.debug("Replacing existing case IAC, caseId: {}", targetCase.getId());
```

Comment:
logging level should be `info` here

Reply:
We tend to only use info at endpoint level. And debug for implementation etc. Or at least that is how it is in the python services.

Reply:
I think thats changing now, this seems like it should be info, but I'm not sure, so will go on your recommendation

Reply:
I don't know we don't seem to have any guidance for the java services so I just went with what we have in Python.